packages/calendar/components/ChannelFilter/index.jsx

```diff
@@ -45,6 +73,7 @@ const ChannelFilter = ({ channelFilter, channelSelected }) => {
     <Wrapper>
       <Select
         onSelectClick={selectedItem => {
+          handleSelectedChannels(channelsSelected.length, channels.length);
```

Comment:
This line will also need to be adjusted to something like this:
```suggestion
          handleSelectedChannels({
            beforeFilterCount: channelsSelected.length,
            afterFilterCount: channels.length,
      });
```